addons/io_scene_gltf2/blender/exp/gltf2_blender_generate_extras.py

```diff
@@ -16,6 +16,37 @@
 import bpy
 from io_scene_gltf2.blender.com import gltf2_blender_json
 
+def to_json_compatible(value):
```

Comment:
please make this private (rename to `__to_json_compatible`) and move it below `generate_extras`